### ISSUE TITLE
gui_rpc_auth.cfg exists bunt cannot read (7.16.15)

### ISSUE THREAD

**Issue by mx75** [BOINC/boinc#4021]
**Describe the bug**
A gui_rpc_auth.cfg (and possibly other) access problem.

**Steps To Reproduce**
1. open new terminal window (under my user, NOT boinc)
2a. type: boincmgr (boincmgr message: "gui_rpc_auth.cfg not found Try reinstalling BOINC")
2b. if run boincmgr on one of remote host, the message is: "gui_rpc_auth.cfg exists but can't be read. Check the file permissions"
3. close boincmgr
4. cd /var/lib/boinc-client (the boinc user's home directory)
5. type boincmgr (at now: run correctly, show projects, tasks, ....)

**Expected behavior**
I want to run boincmgr from anywhere.  (from any directory).

**System Information**
(on all the desktop computers that belong to me):
 - OS: Ubuntu 20.04.1 LTS
 - BOINC Version:  7.16.15+dfsg.is.7.16.11+dfsg+202009010931~ubuntu20.04.1

-- 
mailto:michael@telkp.bme.hu

**Comment by eMPee584** [comment]
grepping `journalctl -u boinc-client.service` revealed it.. Commit 7883508131069ff9c12f086264ba34249b3b0c9b removed support for empty passwords, breaking the common (and arguably legitimate) usage pattern of allowing unauthenticated access if the sysadmin in charge decides so. Before:

```
gui_rpc_auth.cfg is empty - no GUI RPC password protection
```
After:
```
gui_rpc_auth.cfg is empty - assigning new GUI RPC password
Can't open gui_rpc_auth.cfg - fix permissions
```

**Comment by AenBleidd** [comment]
Closed as a duplicate of #4267
